Provisioners/Nginx/Debian/Requirements

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exit 0
```

Comment:
don't you think we have to test if requirements installed by prelude are meet?
generally speaking, we should
- export a var in `_Prelude` and test the var in each `Requirements` file
- test explicitely each requirement, even if installed by `_Prelude`
  here, it is
  - bats